isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -3050,6 +3195,24 @@ namespace isobus
 						}
 						break;
 
+						case static_cast<std::uint8_t>(Function::AuxiliaryInputStatusTypeTwoEnableCommand):
+						{
+							std::uint16_t inputObjectID = message->get_uint16_at(1);
+							bool shouldEnable = message->get_bool_at(3, 0);
+							bool isInvalidObjectID = true;
+							for (auto &input : parentVT->ourAuxiliaryInputs)
```

Comment:
Should we use `std::find_if` ? I don't have a lot of experience with it personally but situations like this and line 3120 make me think "hmmmm"

Reply:
Isn't `std::find_if` a C++20 feature? https://en.cppreference.com/w/cpp/algorithm/find

Reply:
Although it's definitely in 20, it seems like there might be a version prior as well, as theres a "until C++20" on one of them?
![image](https://user-images.githubusercontent.com/10929341/223749116-369b0648-b72b-4f1f-95a8-6a195652ac8c.png)

Reply:
Ah, I see. I like `find_if` more now I used it a couple of times 😄